### PR TITLE
Fix: README.md grammar

### PR DESCRIPTION
**Pull request by DalpatRathore**
Hi! 
&nbsp;&nbsp;&nbsp;&nbsp; I have fixed a few grammatical mistakes and missing article `the` in the `README.md` file. Please review this contribution.

Thanks!

### CODE REVIEW

**Comment by DalpatRathore** [comment]
@mjbvz :confused:  about what:question:

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Many of the core components and extensions to VS Code live in their own reposito
 
 ## Bundled Extensions
 
-VS Code includes a set of built-in extensions located in the [extensions](extensions) folder, including grammars and snippets for many languages. Extensions that provide rich language support (code completion, Go to Definition) for a language have the suffix `language-features`. For example, the `json` extension provides coloring for `JSON` and the `json-language-features` extension provides rich language support for `JSON`.
+VS Code includes a set of built-in extensions located in the [extensions](extensions) folder, including grammar and snippets for many languages. Extensions that provide rich language support (code completion, Go to Definition) for a language have the suffix `language-features`. For example, the `json` extension provides the coloring for `JSON` and the `json-language-features` extension provides rich language support for `JSON`.
```
Comment (gjsjohnmurray) [inline README.md:56]: Incorrect change of grammars to grammar. Plural is needed in this context.

Added 'the' is redundant.

**Review by gjsjohnmurray — COMMENTED** [review]
IMO this PR is a poor use of core team resources.

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
 
-This repository ("`Code - OSS`") is where we (Microsoft) develop the [Visual Studio Code](https://code.visualstudio.com) product together with the community. Not only do we work on code and issues here, we also publish our [roadmap](https://github.com/microsoft/vscode/wiki/Roadmap), [monthly iteration plans](https://github.com/microsoft/vscode/wiki/Iteration-Plans), and our [endgame plans](https://github.com/microsoft/vscode/wiki/Running-the-Endgame). This source code is available to everyone under the standard [MIT license](https://github.com/microsoft/vscode/blob/main/LICENSE.txt).
+This repository ("`Code - OSS`") is where we (Microsoft) develop the [Visual Studio Code](https://code.visualstudio.com) product together with the community. Not only do we work on code and issues here, but we also publish our [roadmap](https://github.com/microsoft/vscode/wiki/Roadmap), [monthly iteration plans](https://github.com/microsoft/vscode/wiki/Iteration-Plans), and our [endgame plans](https://github.com/microsoft/vscode/wiki/Running-the-Endgame). This source code is available to everyone under the standard [MIT license](https://github.com/microsoft/vscode/blob/main/LICENSE.txt).
```
[the remark this answers is not in the document]
Comment (DalpatRathore) [reply README.md:8]: # Prepositional Phrase
vscode-grammar-1

File: README.md (hunk 1 above)
Comment (DalpatRathore) [reply README.md:56]: # Words in Phrase
vscode-grammar-2
# Article
vscode-grammar-3
# Determiner
vscode-grammar-4

**Comment by DalpatRathore** [comment]
Hey! @gjsjohnmurray, I haven't concocted these rules for mere contribution. These grammar rules exist and the references are taken from **grammarly** and one can also refer to the grammar textbook. 

This contribution **IF** approved want make any breaking changes rather it makes the **VS Code** documentation more robust. 

My bit is done with contribution and clarification, feel free to make your case for REJECTION or APPROVAL.

**Comment by mjbvz** [comment]
@gjsjohnmurray is spot on here

The changes themselves are questionable and there's already been far too much time spent arguing over grammar rules

**Comment by DalpatRathore** [comment]
```mermaid
flowchart TD
    A(Contribution) 
    A--> B[Confused Emoji] 
    B-->C(Confused About What ??)
    C--> D[From somewhere IMO jumps in to give IMO]
    D-->E(IMO rebutted with facts & standard rules)
    E-->F{REJECTED}
    F-->G((Why??))
    G-->H[IMO matter]
    G-->I[Standard rules don't matter]
    H-->J((END OF STORY))
    I-->J((END OF STORY))
```

**Pull request closed without merging**